scripts/check_docs_formatted.py

```diff
@@ -16,12 +17,21 @@
     from collections.abc import Sequence
 
 SNIPPED_RE = re.compile(
-    r"(?P<before>^(?P<indent> *)```(?:\s*(?P<language>\w+))?\n)"
+    r"(?P<before>^(?P<indent>\x20*)```(?:\s*(?P<language>\w+))?\n)"
     r"(?P<code>.*?)"
     r"(?P<after>^(?P=indent)```\s*$)",
     re.DOTALL | re.MULTILINE,
 )
 
+# https://www.rexegg.com/regex-best-trick.html
```

Comment:
I'm not sure what this comment is telling me and it's also a rather long article. It would be good to write a short summary here of what trick you apply here. You can still link to the website for the curious once.